### PR TITLE
HW#2-Viktoriia-Bortnikova

### PR DESCRIPTION
**Pull request by Masakinka**
@dimigor

### CODE REVIEW

**Comment by Masakinka** [comment]
[HW#2 - OOP_Practice - Bortnikova Viktoriia](https://trello.com/c/9t0Xl5V0/43-hw2-ooppractice-bortnikova-viktoriia)

**Comment by Masakinka** [comment]
[HW#2 - OOP_Practice - Bortnikova Viktoriia](https://trello.com/c/9t0Xl5V0/43-hw2-ooppractice-bortnikova-viktoriia)

**Comment by Masakinka** [comment]
[HW#2 - OOP_Practice - Bortnikova Viktoriia](https://trello.com/c/9t0Xl5V0/43-hw2-ooppractice-bortnikova-viktoriia)

**Comment by Masakinka** [comment]
[HW#2 - OOP_Practice - Bortnikova Viktoriia](https://trello.com/c/9t0Xl5V0/43-hw2-ooppractice-bortnikova-viktoriia)

File: HW#2/Person.py [hunk 1]
```diff
@@ -102,4 +103,4 @@ def make_purchase(self, discount=0.1):
         person.about_yourself()
         person.make_money()
         person.buy_house()
-        person.make_purchase()
+        person.make_purchase()
```
Comment (dimigor) [inline HW#2/Person.py]: не забувай про endline

File: venv/HW#2/Person.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
+"""
+This is a class Person.
```
Comment (dimigor) [inline venv/HW#2/Person.py]: це модуль)

File: venv/HW#2/Person.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+"""
+from abc import ABC, abstractmethod
+import datetime
+import random
+
+
+# classes of Human
+class Human(ABC):
+
```
Comment (dimigor) [inline venv/HW#2/Person.py]: так само як модуль, продокументуй клас

**Review by dimigor — CHANGES_REQUESTED** [review]

**Comment by Masakinka** [comment]
[HW#2 - OOP_Practice - Bortnikova Viktoriia](https://trello.com/c/9t0Xl5V0/43-hw2-ooppractice-bortnikova-viktoriia)

**Review by dimigor — APPROVED** [review]

**Pull request merged**